### PR TITLE
Fix name of cache key column to be all lower case rather than camel c…

### PR DESCRIPTION
**Pull request by seamuslee001**
…ase in civicrm_prevnext_cache

Overview
----------------------------------------
This renames the cacheKey column to be cachekey to be more standard with other column names

Before
----------------------------------------
camelCase column name used

After
----------------------------------------
lower case used

ping @eileenmcnaughton @colemanw

### CODE REVIEW

File: CRM/Contact/Page/AJAX.php [hunk 1]
```diff
@@ -733,7 +733,7 @@ public static function getDedupes() {
       $join .= " LEFT JOIN civicrm_address ca1 ON (ca1.contact_id = pn.entity_id1 AND ca1.is_primary = 1 )";
       $join .= " LEFT JOIN civicrm_address ca2 ON (ca2.contact_id = pn.entity_id2 AND ca2.is_primary = 1 )";
     }
-    $iTotal = CRM_Core_BAO_PrevNextCache::getCount($cacheKeyString, $join, $whereClause, '=', $queryParams);
+    $iTotal = CRM_Core_BAO_PrevNextCache::getCount($eacheKeyString, $join, $whereClause, '=', $queryParams);
```
Comment (eileenmcnaughton) [inline CRM/Contact/Page/AJAX.php]: opps

Comment (seamuslee001) [reply CRM/Contact/Page/AJAX.php]: Fixed

**Comment by eileenmcnaughton** [comment]
Looks good - I'll let jenkins weigh in before I dig deeply

**Comment by seamuslee001** [comment]
Test failures appear to relate will investigate shortly

**Comment by seamuslee001** [comment]
@eileenmcnaughton i think i have fixed the Jenkins failures now

**Comment by eileenmcnaughton** [comment]
@seamuslee001 this looks good to me - let's merge if tests pass & then I'll do some more work on dedupe cleanup after which should put it through it's paces even more

**Comment by eileenmcnaughton** [comment]
Yay @seamuslee001 that will save future pain!

**Comment by colemanw** [comment]
@totten - FYI I know you've done a lot of work in the cache system, so if you or @eileenmcnaughton have any extensions e.g. for the WMF caching project that query this table, they will need to be updated.

**Comment by eileenmcnaughton** [comment]
@colemanw thanks for thinking of that but no that should not apply

**Pull request merged**